### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by Anzo52** [Anzo52/concalcjava#4]
Improve documentation

**Comment by mikeysan** [comment]
Actually, to do any sort of justice to the documentation we need more information; screenshots, how-to run, etc.

**Comment by Anzo52** [comment]
Agreed. I added a bit more info to the readme, which should hopefully help a bit. I'm trying to improve my knowledge of best practices for architecture and documentation, frankly I'm severely lacking in that area.

**Comment by Anzo52** [comment]
> I don't know Java but I could help improve the documentation a bit.

Just take away the type declarations and curly-brackets and it's basically python. ;-)

**Comment by Anzo52** [comment]
@mikeysan If you're still able to contribute, go ahead and start a fork and link this issue in the PR. I'll hopefully have issue templates up soon. Thanks!

**Comment by mikeysan** [comment]
While I am willing, I am unable to at this time. :(

**Comment by Anzo52** [comment]
Understandable. Would you like me to un-assign it?
